tools/sdk-cli/src/main.rs

```diff
@@ -82,3 +80,40 @@ fn show_results(res: Result<String>) {
         Err(err) => error!("Error: {}", err),
     }
 }
+
+fn init_logging() {
```

Comment:
I was thinking if the sdk logs to a file by default then the cli doesn't need to configure anything and we get the log file for any user of the sdk (not just the cli).

Reply:
Good idea, done